new-chatbot-app/src/library-api/libcal-authorization/libcal-authorization.service.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { LibcalAuthorizationService } from './libcal-authorization.service';
+
+describe('LibcalApiService', () => {
+  let service: LibcalAuthorizationService;
+
+  beforeEach(async () => {
+    const module: TestingModule = await Test.createTestingModule({
+      providers: [LibcalAuthorizationService],
+    }).compile();
```

Comment:
This should be the same as the imports and providers in the library-api.module.ts. So it should be 
const module: TestingModule = await Test.createTestingModule({
      imports: [
    SharedModule,
    HttpModule
  ],
  providers: [
    LibcalAuthorizationService,
  ]
    }).compile();